app/forms/webauthn_verification_form.rb

```diff
@@ -64,12 +76,12 @@ def self.domain_name
 
   def validate_assertion_response
     return if webauthn_error.present? || webauthn_configuration.blank? || valid_assertion_response?
-    errors.add(:authenticator_data, 'invalid_authenticator_data', type: :invalid_authenticator_data)
+    errors.add(:authenticator_data, :invalid_authenticator_data, message: generic_error_message)
```

Comment:
This inverts the "type" and "message", where in most other code we'd do the following:

```suggestion
    errors.add(:authenticator_data, generic_error_message, type: :invalid_authenticator_data)
```

As mentioned in #9569, I think the pattern in the suggested code above is not really expected to be supported by [Rails' `ActiveModel::Errors#add` method](https://api.rubyonrails.org/classes/ActiveModel/Errors.html#method-i-add), which is documented as receiving `type` as the second parameter, and optionally a custom `message` option. The code proposed here follows the documented usage, and I'd be interested to consider porting existing code to this pattern.